gke-deploy/cloudbuild.yaml

```diff
@@ -0,0 +1,16 @@
+steps:
+# Run tests.
+- name: 'golang:stretch'
+  entrypoint: 'go'
+  args: ['test', './...']
+
+# Build the gke-deploy binary and put into the builder image.
+- name: 'gcr.io/cloud-builders/docker'
+  args: ['build', '-t', 'gcr.io/$PROJECT_ID/gke-deploy', '.']
+- name: 'gcr.io/$PROJECT_ID/gke-deploy'
+  args: ['--help']
+
+images:
+- 'gcr.io/$PROJECT_ID/gke-deploy'
+
+timeout: '2100s'
```

Comment:
Is this needed? How long does this normally take?

Reply:
This was a remnant from when I had a functional test here before. I have now removed it.